client/components/cloud/project/billing/outgoing-trafic/outgoing-trafic.component.js

```diff
@@ -0,0 +1,9 @@
+const OutgoingTraficComponent = {
```

Comment:
Use JS modules

Reply:
Do you mean ES6 modules? can you please give me an example

Reply:
Sure, look at this: https://github.com/ovh-ux/ng-ovh-telecom-universe-components/blob/master/src/telecom/voip/index.js

Reply:
These changes will be made as part of cloud migration to One Manager right?

Reply:
You can start organizing your code with modules right now :) this would :
* ease the work of our teams working on the One Manager as they will have less to do
* make us learn a better way to structure our code base as soon as possible :)

Reply:
As this task has been validated by QC I will merge it in order to be able to start the translations request.

But feel free to open a separate pull request to handle some refactoring such as using the module approach for instance.